configs/example_configs_validation.py

```diff
@@ -0,0 +1,21 @@
+
+import os
+import subprocess
+import sys
+
+
+def main():
+    for arg in sys.argv[1:]:
+        if not arg.endswith(".yaml"):
+            continue
+        result = subprocess.run(f"tools/config_validation/validate_fragment envoy.config.bootstrap.v3.Bootstrap {arg}".split())
```

Comment:
You could pass `envoy.config.bootstrap.v3.Bootstrap` as a parameter, and then have a separate target for  DisocveryResponse validation. Or meybe even group type and paths and validate everything in one target...

Reply:
i think these targets are all currently bootstrap - but it also includes certs etc

ill separate these out and look at any additional targets - like other types of config file